src/View/StringTemplate.php

```diff
@@ -369,4 +369,46 @@ public function addClass($input, $newClass, $useIndex = 'class')
 
         return $input;
     }
+
+    /**
+     * Removes a class and returns a unique list either in array or space separated
+     *
+     * @param array|string $input The array or string to remove the class from
+     * @param array|string $removeClass the new class or classes to remove
```

Comment:
array could be string[] here, right?

Reply:
Yep, I think if it's an array, it's supposed to only be string[].. The `addClass()` one seems pretty flexible and intuitive, and `removeClass()` is using the same methodology

Reply:
Just verified that that it does work like `addClass()` in that it supports both string and string[], I do need to add official test cases though